gluoncv/model_zoo/rcnn/roi_heads.py

```diff
@@ -0,0 +1,23 @@
+"""ROI heads."""
+
+from __future__ import absolute_import
+
+from mxnet.gluon import nn
+
+
+class MLPFeatureExtractor(nn.HybridBlock):
+    """
+    Head for FPN for classification and box regression.
```

Comment:
missing doc for arguments

Reply:
okay.. this is not supposed to be here...